docs/sdk/v3/guides/liquidity/03-checking-collecting-fees.md

```diff
@@ -1,6 +1,57 @@
 ---
 id: liquidity-fees
-title: Checking & Collecting Fees
+title: Collecting Fees
 ---
 
-TODO
\ No newline at end of file
+## Introduction
+
+This guide will cover how to collect fees from a liquidity position on the Uniswap V3 protocol.
+It is based on the [collecting fees code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/collecting-fees), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/v3-sdk/collecting-fees/README.md) and follow the setup instructions.
+
+:::info
+
+If you need a briefer on the SDK and to learn more about how these guides connect to the examples repository, please visit our [background](./01-background.md) page!
+
+:::
+
+In the Uniswap V3 protocol, liquidity positions are represented using non-fungible tokens. In this guide we will use the `NonfungiblePositionManager` class to help us mint a liquidity position for the  **USDC - DAI** pair. We will then attempt to collect any fees that the position has accrued from those trading against our provisioned liquidity. The inputs to our guide are the **two tokens** that we are pooling for, the **amount** of each token we are pooling for, the Pool **fee** and the **max amount of accrued fees** we want to collect for each token.
+
+The guide will **cover**:
+1. Collecting accrued fees from our position
```

Comment:
It feels weird to have a single step list. Can we either break down into two steps (e.g. creating the options and executing the transaction) or make this a single section?